cmake/BLTMacros.cmake

```diff
@@ -663,3 +663,56 @@ macro(blt_append_custom_compiler_flag)
 
 endmacro(blt_append_custom_compiler_flag)
 
+
+
+
+##------------------------------------------------------------------------------
+## blt_find_libraries( <FOUND_LIBS>
```

Comment:
Sorry I wasn't more clear.  This needs to be changed to:

blt_find_libraries(FOUND_LIBS <FOUND_LIBS>

Unnamed variables are a large hassle to parse if someone wants to wrap this macro.